pkg/types/command.go

```diff
@@ -189,5 +185,5 @@ func (c *Command) ToCmdline() []string {
 // Cmd converts Command to a commandline format and returns an exec.Cmd which
 // can be executed by os/exec
 func (c *Command) Cmd(gvproxyPath string) *exec.Cmd {
-	return exec.Command(gvproxyPath, c.ToCmdline()...)
+	return exec.Command(gvproxyPath, c.ToCmdline()...) // nolint:gosec
```

Comment:
Why is this ok/harmless to ignore? Could you add an explanation in the commit log?

Reply:
Updated, but I'll post it here too.

 "If `gosec` linting is enabled for the return statement in `Cmd`, an error will be returned: `G204: Subprocess launched with a potential tainted input or cmd arguments (gosec)`. This error tries to make sure a user cannot provide a binary that could cause harm to the system. However, since the binary path is sanitized and the arguments are generated by gvproxy, this should be safe to ignore. "
 
 I'm not sure if this is the best course of action, so if there are any other ideas I'd be happy to go about it in another direction.